docs/DeveloperGuide.md

```diff
@@ -395,8 +395,29 @@ otherwise)
 
 ### Glossary
 
+* **AddressBook**: A personal management tool used to store contact details, such as names and phone numbers
+* **API(Application Programming Interface)**: Defines how software components interact with each other
+* **Architecture**: The high-level design and structure of the application consisting of components such as UI, Model and Storage
+* **CLI (Command Line Interface)**: A user interface that is based on interaction with the terminal or console
+* **Commons**: A collection of shared resources or classes used by multiple parts of the application
+* **Command**: An object representing a user commend, created by parsing user input.
```

Comment:
Perhaps we could standardize the lack of the full stops at the end of sentences in this section?
```suggestion
* **Command**: An object representing a user commend, created by parsing user input
```

Reply:
okay